### PR TITLE
Clarifying that Consultants cannot be practitioners

### PR DESCRIPTION
**Pull request by jrstorment**
The definition of consultants / vendors is ambiguous, aiming to clarify it for membership purposes.

### CODE REVIEW

**Comment by jrstorment** [comment]
This should likely go to TAC for debate, feedback and ultimately a vote for approval.

File: practitioner_definition.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 This file outlines the criteria for joining the FinOps Foundation as a Practitioner member.
 
-> Practitioners must: (i) be actively working in FinOps: (a) in an executive capacity, such as a Chief Information Officer or Chief Technology Officer; (b) in IT/engineering and operations; (c) in finance and/or procurement; (d) as a FinOps practitioner; or (e) in a similar capacity (regardless of title); (ii) not be employed by a FinOps, cloud, or cloud management vendor; and (iii) not have FinOps-related sales, marketing, or business development as their primary business. 
+> Practitioners must: (i) be actively working in FinOps or cloud financial management related work: (a) in an executive capacity, such as a Chief Information Officer or Chief Technology Officer; (b) in IT/engineering and operations; (c) in finance and/or procurement; (d) as a FinOps practitioner; or (e) in a similar capacity (regardless of title); (ii) not be employed by a FinOps, cloud, or cloud management vendor or company offering consultancy or advisory services; and (iii) not have FinOps-related sales, marketing, or business development as their primary business, regardless of the company they are in.
```
Comment (arottem) [inline practitioner_definition.md:5]: Seems a bit wordy.
As "financial operations" is abbreviated to FinOps (with no "cloud" ahead of it), then the opposite, fully spelling out "cloud financial management".
How about compromise:
"working in financial operations or management related work"

Comment (tfcross) [reply practitioner_definition.md:5]: As someone who works in a team of a practitioners but as part of a wider company that also offers consultancy and advisory services, I'm not sure how I feel about this amendment as it would appear to mean that I can't be considered a practitioner!

Comment (casaubon42) [reply practitioner_definition.md:5]: I find this restrictive definition of "practitioner" (old and new) quite confusing...
Some consulting and cloud management firms are already partners with the FinOps Foundation (e.g. my company timspirit), which means you could potentially ban your own partner... 
I am myself an employee of a "company providing consulting or advisory services", and as such, I have to keep up to date with FinOps best practices when doing FinOps on behalf of my client (like any other non-contractor practitioner). And the F2 is a great way to do that and i use it honestly.
imho, I think we should focus on keeping the F2 neutral with a strict code of conduct for members, let everyone interesting joining in and eventualy cap some SIG or WG members to a specific population (e.g limit members from CSP in the Educaiton SIG to 20%) if needed.
Personnaly, I have already been "pitched" by vendors on Slack by PM : we should mainly take care to enforce the "rules" afterwards.

Comment (jrstorment) [reply practitioner_definition.md:5]: Thanks everyone for the input, clearly we need to have some more discussion on this topic. 

To be clear: we have no intention of excluding consultants from participating in the community. We are examining our approach to consultants based on confusion/feedback from the industry and this is a first step in separating out different types of members more clearly, with the goal of actually INCREASING consultant participation, not limiting it.

With the change above, we were looking to clarify and categorize different types of members who have different roles, responsibilities and ways of interacting with the Foundation. 

The original spirit of the Practitioner definition did not include them but this was not clear enough. It did not include them because the GB felt there was value in being able to categorize "those who do FinOps at a consumer of cloud as their day job (e.g., a Fidelity)" vs. "those who sell services to consumers of cloud (e.g., a Cloudreach)". 

Keep in mind, we do have paying programs for them to participate and certify (e.g., FinOps Certified Service Provider). Currently the free practitioner experience is subsidized by those programs and our Governing Board wants to keep things free for practitioners if at all possible, which means we need vendors (technology or consulting) to help cover costs.

The GB is working on a larger proposal for consultants that we'll socialize before we nail down the above proposed change. Stay tuned.
